### DISCUSSION TITLE
How to dynamically update select options

### DISCUSSION THREAD

**Discussion by darren-glanville** [filamentphp/filament#10033] [Help] [answered]
I have a dropdown that I would like to update the options based on when another field is changed, how do I go about updating it. 

I would like to run a Closure is an ideal world.

**Comment by ArtMin96** [comment, marked as answer]
[Dependant select options](https://filamentphp.com/docs/3.x/forms/advanced#dependant-select-options)

**Comment by leandrocfe** [comment]
Share your code please
